src/main.ts

```diff
@@ -11,6 +11,7 @@ async function bootstrap() {
         .build();
     const document = SwaggerModule.createDocument(app, options);
     SwaggerModule.setup('api', app, document);
+    app.enableCors();
```

Comment:
do we have another plan instead of Cross-origin?

Reply:
This is necessary or else frontend won't be able to access API. In the future, maybe we should whitelist just some URL for security.